package.json

```diff
@@ -84,7 +80,7 @@
     "fs-extra": "10.1.0",
     "get-port": "5.1.1",
     "glob": "7.2.3",
-    "husky": "3.1.0",
+    "husky": "^8.0.1",
```

Comment:
inline with the rest of the dependencies this should be locked please.

Reply:
pinned the husky version